packages/main/src/plugin/kubernetes/contexts-manager.ts

```diff
@@ -76,7 +76,7 @@ import { ResourceWatchersRegistry } from './resource-watchers-registry.js';
 // only the connectivity to the current context will be checked
 const MAX_NON_CURRENT_CONTEXTS_TO_CHECK = 10;
 
-const kindsWithEvents = ['Deployment', 'Pod'];
+const kindsWithEvents = ['Deployment', 'Pod', 'Node'];
```

Comment:
variable not uppercase ? (already like that before this PR so not blocking)

Reply:
changed to uppercase.

Also made a change based on @benoitf review at https://github.com/containers/podman-desktop/pull/9664#discussion_r1822270518